kbhff/api/email.py

```diff
@@ -43,12 +66,9 @@ def mail_match(mail):
     for i in range(0,retry_count):
         if len(matches) > 0:
             break
-        time.sleep(1)
+        time.sleep(10)
```

Comment:
There's already a sleep/retry loop built in to the `GmailConnection` class; it would probably be better to let it handle the retrying. (If the issue is that sometimes `listup` returns without an error but erroneously returns an empty list, then you could add a parameter `wait_for_email` that will also keep retrying if there are no emails returned.)